scripts/dts/edtlib.py

```diff
@@ -134,6 +134,10 @@ class EDT:
       A collections.OrderedDict that maps a node label to the node with
       that label.
 
+    depord2node:
```

Comment:
Can we come up with a better name?  Or dep_ord2node?

Reply:
I don't care what it's named.

Reply:
Rebased and renamed `dep_ord2node`.